build-config/scripts/onie-mk-itb.sh

```diff
@@ -139,6 +141,7 @@ DTB="$(realpath $DTB)"
 			arch = "$ARCH";
 			os = "linux";
 			compression = "none";
+			load = <$FDT_LOAD>;
```

Comment:
OK, it seems we should make the `load = <$FDT_LOAD>;` line conditional -- only include it for the machines that really need it.

To do that we could:

- by default set `FTD_LOAD` to the string `no` using `?=`.  That should be done in `arch/{powerpc,armv7,armv8}.make`, right after setting `KERNEL_ENTRY_POINT`.

- update make/images.make to pass `$(FTD_LOAD)` to the `onie-mk-itb.sh` script

- inside `onie-mk-itb.sh` check whether `$FTD_LOAD` is equal to `no`.  If it is, then do nothing.  Otherwise use the value of `$FTD_LOAD` to add the `load = $FTD_LOAD` line to the dtb stanza.

- for machines that need FTD_LOAD, it in their `machine.make` file.

Something like that would do it I think.